### PR TITLE
Improve license headers

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Improve license headers
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | 
| Possible impacts? | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27070)

### CODE REVIEW

File: tests/Unit/PrestaShopBundle/Command/DebugCommandTest.php [hunk 1]
```diff
@@ -1,4 +1,29 @@
...
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
```
Comment (matthieu-rolland) [inline tests/Unit/PrestaShopBundle/Command/DebugCommandTest.php:26]:
```suggestion
```

File: tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
```
Comment (matthieu-rolland) [inline tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php:26]:
```suggestion
```

File: tests/Integration/Behaviour/Features/Context/Util/StreamWrapperPHP.php [hunk 3]
```diff
@@ -1,4 +1,29 @@
...
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
```
Comment (matthieu-rolland) [inline tests/Integration/Behaviour/Features/Context/Util/StreamWrapperPHP.php:26]:
```suggestion
```

**Review by matthieu-rolland — COMMENTED** [review]
I made some code suggestions, I think it's what breaks cs fixer 🤔

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**